SEDaily-IOS/PodcastCollectionViewCell.swift

```diff
@@ -68,7 +68,12 @@ class PodcastCell: UICollectionViewCell {
 
     private func setupTimeDayLabel(timeLength: Int?, date: Date?) {
         let dateString = date?.dateString() ?? ""
```

Comment:
This might be better off being it's own label or renamed to "misc details". Tried to think of a nice icon to use but I couldn't find any that actually got this concept across.